hello-world-microservice/src/main/java/c8y/example/helloworld/GaugeStorage.java

```diff
@@ -0,0 +1,22 @@
+package c8y.example.helloworld;
+
+import io.opentelemetry.api.metrics.ObservableDoubleGauge;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class GaugeStorage {
```

Comment:
This component stores the list of gauge callbacks to be closed post export. See the comment [mentioned in PR ](https://github.com/SoftwareAG/cumulocity-clients-java/pull/273/files#r1132325035) for the explanation